pkg/config/linters.go

```diff
@@ -9,3 +14,52 @@ type Linters struct {
 
 	Presets []string
 }
+
+func (l *Linters) Validate() error {
+	if err := l.validateAllDisableEnableOptions(); err != nil {
+		return err
+	}
+
+	if err := l.validateDisabledAndEnabledAtOneMoment(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (l *Linters) validateAllDisableEnableOptions() error {
+	if l.EnableAll && l.DisableAll {
+		return errors.New("--enable-all and --disable-all options must not be combined")
+	}
+
+	if l.DisableAll {
+		if len(l.Enable) == 0 && len(l.Presets) == 0 {
+			return errors.New("all linters were disabled, but no one linter was enabled: must enable at least one")
+		}
+
+		if len(l.Disable) != 0 {
+			return fmt.Errorf("can't combine options --disable-all and --disable %s", l.Disable[0])
```

Comment:
proposal: make consistent with govet, gocritic, something else? errors (remove `Disable[0]`, change text)

Reply:
It's a different topic (golangci-lint vs linters).
I need to think about it :thinking: 
We can handle it in a separate PR if needed.